TestAssets/TestProjects/TestProjectWithCultureSpecificResource/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is there an explicit reason this has its own NuGet.Config?  I have an issue to remove these:  https://github.com/dotnet/cli/issues/1316.  Can you remove it if it isn't needed?